### ISSUE TITLE
Can't track FL Blue Pyrite

### ISSUE THREAD

**Issue by Ninjaskill** [Bunny67/WeakAuras-WotLK#43]
Forced to use legacy auras to track it for myself. Non-legacy won't track it if you check "own only"

![image](https://user-images.githubusercontent.com/10984882/90313118-4942c100-df0a-11ea-843b-dd4ee5d41ec1.png)

**Comment by Bunny67** [comment]
Works for me. Aura and Legacy Aura with the same parameters behave the same

**Comment by Ninjaskill** [comment]
Me and guildies cannot get WA to work to track Blue Pyrite for just "yourself". When I uncheck "own only" it works but globally. When I check "own only" it doesn't work. If I do legacy aura with checking "own only" it works.

**Comment by Bunny67** [comment]
Yes, when you are in a vehicle, the unit target does not work correctly. In retail same behavior.

**Comment by Sanaka3** [comment]
there's a workaround for that if you don't mind using auto-clone

that workaround is playing around conditions, making it visible (alpha to 100%) with If trigger 1 (or which one checks pyrite stacks) -> caster -> is exactly -> vehicle. this is what I use to highlight my stacks, since all buffs/debuffs are in there, but with color change

though for that you'll have to also mess around with how it will look. that, or try making it work with custom code instead

this "own only" parameter (I think) should be made to work with vehicles, if possible. hope it all helps

**Comment by Ninjaskill** [comment]
Thanks for the suggestion @Sanaka3 but Joakim on ElvUI - Backport Discord has shared a pretty nice FL WA with custom triggers.
